### DISCUSSION TITLE
Possible blog for future sharing of updates, insights, and overall user interactions

### DISCUSSION THREAD

**Discussion by Andre601** [mkdocs/mkdocs#3664] [Ideas]
MkDocs could benefit from offering a blog, where posts about updates, insights, etc are being shared related to MkDocs.

It could allow more interaction with users, especially when a comment system (Such as giscus) would be included here and would allow the MkDocs team to share insights to the project with users, such as what more recent updates brought to the table.
This could even go as far as to have smaller guides covering some of MkDocs features in more detail that may not be worth having a own user guide page for.

The issues here are a few tho.
First, how would the blog be made? Making it on some 3rd-party wouldn't really be good, but would it be worth to have it in the MkDocs... docs themself using a plugin?
Next, who would maintain it and how should it be structured (Categories, topics, post schedules, etc)?
I would be willing to help out making posts (With some people proof-reading), but we should initially have a decent structure and some ideas to cover things for the future...

Overall, I think it could be an interesting idea to have here.

**Comment by pawamoy** [comment]
I think [Material for MkDocs' blog plugin](https://squidfunk.github.io/mkdocs-material/plugins/blog/) has us covered, if we want to use it for MkDocs' docs :wink: 
Consider Textual's integrated blog as an example: https://textual.textualize.io/blog/.
I'm imagining publishing posts from different authors (after review by our team of course), and that looks gorgeous in my mind :heart_eyes:

**Reply by Andre601** [reply]
Yep. The blog plugin is very well made (I use it too).

Tho, it would need work, given that MkDocs' site is a different theme, so it would need some integration (Through templating I think), to work, unless I'm missing something.

Either way, another thing the blog would be great for is to further include the community, by having something such as a "Developer" or "Project" spotlight, where a dev or project working with MkDocs would be covered.
It could give the dev/project exposure and connect MkDocs more with the community, by working with the dev in question on that post.

**Reply by pawamoy** [reply]
Definitely, I like that idea.

What I was suggesting is that if we use Material for MkDocs for the MkDocs website, then MM's blog plugin will be fully integrated. But given @squidfunk made efforts to make his plugins modular/standalone, it's maybe possible to use the blog plugin without MM itself (I'll let him tell us).

**Reply by Andre601** [reply]
Using the material theme wouldn't be a good idea, as it would create confusion amongst users ("Why is my docs using a different theme than mkdocs?").

**Reply by pawamoy** [reply]
This is one concern indeed. Some discussion happened in the "New default theme" thread already :) (on mobile, will cross link later)

**Reply by squidfunk** [reply]
Thanks, @Andre601 – I'm happy you like our blog plugin! Getting started with the Material for MkDocs blog plugin is a 5min job. However, I think before we decide on whether MkDocs wants and needs a blog, we should decide on who wants to maintain it and who has the time to write posts 😉 Regarding the confusion part, I already answered in https://github.com/mkdocs/mkdocs-team-space/discussions/4#discussioncomment-9052469.

**Reply by Andre601** [reply]
I was already offering myself for the job of doing the blog.

The only things that I would need help with is:

- Finding topics to make
- Have a somewhat consistent schedule for posts (how frequent)
- General proofreading to ensure proper tone and grammar within the blog post.

Regarding topics can I think of a few ideas:

- Release posts, explaining the most recent MkDocs release in more detail (Should probs be limited to larger/major updates rather than also covering patches, unless we would combine multiple).
- Posts covering certain features of MkDocs in a lot more detail, which would be too much to be part of the "User Guides" section (or "Dev Guides" if it was about a dev-related thing). Maybe even cover some edge/unique cases of using MkDocs
- A "Developer Spotlight" and/or "Project Spotlight" where a theme/extension/plugin dev or one of their projects would be covered. Could be useful in working with the dev together, by having them provide info.

**Reply by squidfunk** [reply]
Thanks for the suggestions and for offering to work on the blog! I suggest we postpone the blog and focus on the more urgent topics that we need to discuss, i.e., how we organize ourselves, how we streamline the process of working together on the code base, and how we can improve the evergreen content that is MkDocs own documentation. When we did all this, we should definitely consider setting up a blog that talks about ephemeral things, but IMHO, it is not the best invest of our time right now.

**Reply by Andre601** [reply]
Understandable.
